src/main/scala/com/stratio/khermes/helpers/faker/Faker.scala

```diff
@@ -57,6 +58,27 @@ trait FakerGenerator extends AppSerializer {
   def randomElementFromAList[T](list: Seq[T]): Option[T] =
     if (list.nonEmpty) Option(list(Random.nextInt((list.size - 1) + 1))) else None
 
+  //TODO: We should provide more strategies.
+  def listWithStrategy[T](list: Seq[T], strategy: String): Seq[(T, Double)] = {
+    var p = 0.0
+    strategy match {
+      case "default" => p = 0.8
+      case _ => p =0.5
+    }
+    val first: (T, Double) = list.head -> p
```

Comment:
Does it mean that we'll be picking up always the first element, this could be a bit "boring" with pre-set list, would make sense to take a random element to make it the most likely??